src/ariel-os-sensors/src/sample.rs

```diff
@@ -1,3 +1,11 @@
+/// Errors returned when trying to interpret a sensor value.
```

Comment:
"Sample" is better defined in the sensor abstraction definitions (in the crate root):
```suggestion
/// Errors returned when trying to interpret a sample.
```